logstash-core/spec/conditionals_spec.rb

```diff
@@ -398,11 +413,11 @@ def multi_receive(events)
       expect(subject).to be_an(Array)
       expect(subject.length).to eq(2)
 
-      expect(subject[0].get("type")).to eq("original")
-      expect(subject[0].get("cond1")).to eq("true")
-      expect(subject[0].get("cond2")).to eq(nil)
+      expect(subject[1].get("type")).to eq("original")
```

Comment:
Did the order change? Why the change from 0 to 1?

Reply:
@jordansissel yea just a result of us not guaranteeing any order within a batch and the graph execution taking advantage of this fact by not using ordered sets.

Reply:
@andrewvc see above, I already made the order deterministic via `subject.sort! {|a, b| a.get("type") <=> b.get("type")}` :)